Puc/v4p2/Plugin/UpdateChecker.php

```diff
@@ -86,6 +86,7 @@ protected function installHooks(){
 			//Override requests for plugin information
 			add_filter('plugins_api', array($this, 'injectInfo'), 20, 3);
 
+			add_filter('plugin_row_meta', array($this, 'addViewDetailsLink'), 10, 2);
```

Comment:
- If you change 2 to 3 here, the callback will receive an additional argument that contains the plugin header data. That's useful for getting the plugin name.
- Remember to remove this hook in the `removeHooks` method.